shell/gpu/gpu_surface_gl_delegate.h

```diff
@@ -15,6 +15,8 @@ namespace flutter {
 
 class GPUSurfaceGLDelegate : public GPUSurfaceDelegate {
  public:
+  virtual ~GPUSurfaceGLDelegate() override;
```

Comment:
virtual isn't needed here if you have override already, I'm surprised the compiler doesnt complain